sound/soc/sdw_utils/soc_sdw_dmic.c

```diff
@@ -1,14 +1,14 @@
 // SPDX-License-Identifier: GPL-2.0-only
-// Copyright (c) 2020 Intel Corporation
+// Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Pretty sure one should be keeping both copyrights here, rather than replacing the existing one, but hopefully someone with more legal knowledge can comment.